packages/taquito-tzip12/src/tzip12-contract-abstraction.ts

```diff
@@ -179,7 +179,7 @@ export class Tzip12ContractAbstraction {
     const bigmapTokenMetadataId = await this.findTokenMetadataBigMap();
     let pairNatMap;
     try {
-      pairNatMap = await this.context.contract.getBigMapKeyByID<any>(
+      pairNatMap = await this.context.contract.getBigMapKeyByID<{token_info: BigMapId}>(
```

Comment:
Not sure the type is right, should be something like this: {token_info: MichelsonMap<string, string>}